### PR TITLE
Rename WarpXWrappers.(h->H)

### PR DESCRIPTION
**Pull request by ax3l**
All our other header files use `.H`, so renaming this one for consistency.

### CODE REVIEW

**Review by dpgrote — APPROVED** [review]
This is ok. I had intentionally use lower case `h` to distinguish this file since everything in it is meant for `C` and not `C++`.
BTW, I don't see why it is included in `PhysicalParticleContainer.cpp` and `WarpX.cpp`. The includes were added in IWYU but I don't see any dependence.

**Comment by ax3l** [comment]
Ah, we compile them anyway in C++ translation units, thus the `extern "C" {` need.

The extra includes are a good point, will check again why it is pulled in more files now.

File: Source/Particles/PhysicalParticleContainer.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #include "Particles/Pusher/UpdatePosition.H"
 #include "Particles/SpeciesPhysicalProperties.H"
 #include "Particles/WarpXParticleContainer.H"
-#include "Python/WarpXWrappers.h"
+#include "Python/WarpXWrappers.H"
```
Comment (ax3l) [inline Source/Particles/PhysicalParticleContainer.cpp]: Indeed not used here, would remove.

File: Source/WarpX.cpp [hunk 2]
```diff
@@ -29,7 +29,7 @@
 #include "Filter/NCIGodfreyFilter.H"
 #include "Particles/MultiParticleContainer.H"
 #include "Particles/ParticleBoundaryBuffer.H"
-#include "Python/WarpXWrappers.h"
+#include "Python/WarpXWrappers.H"
```
Comment (ax3l) [inline Source/WarpX.cpp]: Indeed not used here, would remove.

**Review by dpgrote — APPROVED** [review]
Looks good to me!

**Pull request merged**